client/Main.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+
+export default class Main extends React.Component {
+  constructor(){
+    super()
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.addEducation = this.addEducation.bind(this)
+    this.state = {
+      fullName: "",
+      title: "",
+      email: "",
+      education: [{institution:""
+      // startDate: "",
+      // endDate:""
+      }]
+    }
+  }
+
+  handleSubmit(event) {
```

Comment:
I think you don't need to .bind(this) for each function if you use arrow function here instead
```javascript
handleSubmit = event => {
  event.preventDefault()
  console.log(this.state)

}
```